styleguide-app/ModuleExample.elm

```diff
@@ -124,7 +124,10 @@ categoryForDisplay category =
 view : Bool -> ModuleExample msg -> Html msg
 view showFocusLink { name, content } =
     Html.div
-        []
+        [ String.replace "." "-" name
+            |> (++) "module-example__"
+            |> Attributes.class
+        ]
```

Comment:
❓ What does this do? I can't see that this does much right now within this codebase, though it's likely that I've missed something. Does it support something external? If so, add a comment?